spec/models/classification_spec.rb

```diff
@@ -539,6 +539,29 @@
     end
   end
 
+  describe "name2tag" do
+    let(:root_ns)   { "/managed" }
+    let(:parent_ns) { "/managed/test_category" }
+    let(:entry_ns)  { "/managed/test_category/test_entry" }
+    let(:parent) { FactoryBot.create(:classification, :name => "test_category") }
+
+    it "creates parent tag" do
+      expect(Classification.name2tag("test_category")).to eq(parent_ns)
+    end
+
+    it "creates tag with name and ns" do
+      expect(Classification.name2tag("test_entry", 0, parent_ns)).to eq(entry_ns)
```

Comment:
as I wrote on #18301, I think this tests a misuse; won't hurt but I suggest also adding the more natural case too:
```suggestion
      expect(Classification.name2tag("test_entry", 0, parent_ns)).to eq(entry_ns)
      expect(Classification.name2tag("test_category", 0, root_ns)).to eq(parent_ns)
```